Copy Prev/Next buttons to bottom of commit page

bryan-ash opened the pull request:
When reviewing a long commit in a PR, it's annoying to have to scroll
back to the top of the page to hit the "Next" button to go to the next
commit. Here we duplicate the Prev/Next buttons and insert them at the
bottom of the page too.

Fixes #1387

## Demo

<img width="693" alt="screenshot 2018-06-21 22 42 24" src="https://user-images.githubusercontent.com/24777/41755271-741773de-75a4-11e8-9181-fcc1c73df633.png">

bfred-it commented:
Let’s add a single `if` on both elements to ensure they exist before doing anything. 

Also needs to be added to the readme and it needs a header comment. 

After that I don’t know if it’s approved yet, it wasn’t discussed. I’ll wait a while before merging.

bryan-ash commented:
@bfred-it thanks for the feedback. 

I've added a header comment, and the guard clause for the case when the buttons aren't present, good call.

I added a reference to the screenshot above to the readme.

Your comment "I don’t know if it’s approved yet, it wasn’t discussed" makes me wonder if there's a process documented somewhere that i should have followed. The [Contributing](https://github.com/sindresorhus/refined-github/blob/master/contributing.md) guide states:

> Suggestions and pull requests are highly encouraged!

It might be a good idea to add a header comment to the example file that the guide links to.

bfred-it commented on readme.md:
```diff
@@ -122,6 +122,7 @@ GitHub Enterprise is also supported. More info in the options.
 - [Collapse or expand a repository's browser file list.](https://user-images.githubusercontent.com/1402241/35480123-68b9af1a-043a-11e8-8934-3ead3cff8328.gif)
 - [Quickly visit a repository's default branch and latest version tag.](https://user-images.githubusercontent.com/1402241/38107328-ccb3fb46-33bb-11e8-9654-23a6410943cc.png)
 - [Open multiple issues at once in your repos.](https://user-images.githubusercontent.com/1402241/38084752-4820b0d8-3378-11e8-868c-a1582b16f915.gif)
+- [Navigate Prev/Next commit from the bottom of a commit](https://user-images.githubusercontent.com/24777/41755271-741773de-75a4-11e8-9181-fcc1c73df633.png)
```
This should probably be in "more shortcuts". Maybe the copy can be:

```
Go to the next or previous commit from the bottom of the commit page.
```

sindresorhus commented:
Looks good :)
